tests/test_add_new_openwrap_partner.py

```diff
@@ -491,7 +491,7 @@ def test_OPENWRAP_SETUP_TYPE_inapp(self, mock_input, mock_load_price_csv,
     #check roadblock type
     self.assertEqual(args[19], 'AS_MANY_AS_POSSIBLE')
     #check bidder code
-    self.assertEqual(args[8], None)
+    self.assertEqual(args[8], ['mypartner'])
```

Comment:
You can update the same method for custom targeting as well. In this test case currently, custom targeting is not passed and set.